src/v/cluster/rm_stm.cc

```diff
@@ -3270,6 +3270,12 @@ void rm_stm::setup_metrics() {
     _metrics.add_group(
       prometheus_sanitize::metrics_name("tx:partition"),
       {
+        sm::make_gauge(
+          "idempotency_num_pids_known",
+          [this] { return _log_state.seq_table.size(); },
+          sm::description("Number of known pids in seq_table."),
```

Comment:
nit: since this is user facing, reword a bit to not have implementation specifics?

Reply:
good point, updated